feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -254,12 +264,15 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 		}
 	}
 	communitySetReject.SetCommunityMember(cs1)
-	communitySetReject.SetMatchSetOptions(matchAny)
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
```

Comment:
See above, this doesn't appear to do what the intent of the deviation is as written.